validators/terravaloper1ccwgk6gvgtm556gxe0v79p48nrgsey05w3fle4/README.md

```diff
@@ -0,0 +1,44 @@
+---
+Moniker: Luna Station 88
+Validator: terravaloper1ccwgk6gvgtm556gxe0v79p48nrgsey05w3fle4 
+---
```

Comment:
Email is required for the validator profile. Could you add the email?

Reply:
The email is in the profile.json file, added it to the text too